### DISCUSSION TITLE
Mailer_Email custom FROM name

### DISCUSSION THREAD

**Discussion by rafalkukla** [plausible/analytics#2349] [Self-Hosted Support]
Hiya, 
Is it possible to set Mailer Email name inside .env SMTP configuration? At the moment, email is being sent as hello@domain.com but it would be nice to have option to set FROM name. Thanks

**Comment by ruslandoga** [comment]
👋 According to https://github.com/plausible/analytics/blob/1dcfbf9f903c9607cd73aede4dc3c2c8bfdf83dc/lib/plausible_web/email.ex#L309 and https://github.com/plausible/analytics/blob/1dcfbf9f903c9607cd73aede4dc3c2c8bfdf83dc/config/runtime.exs#L77 you can set a custom FROM with `MAILER_EMAIL`

**Reply by rafalkukla** [reply]
Hi, thanks. But Sorry, still not sure how to view this. 

My understanding is, it does only accept FROM email address, but without FROM NAME like company name?  
How would you define this line below to add FROM "Company Name" as well as email? 

MAILER_EMAIL=Acme Company <hello@domain.com> ?

**Reply by ruslandoga** [reply]
Maybe `MAILER_EMAIL="Acme Company <hello@domain.com>"` :)

**Reply by rafalkukla** [reply]
Ahh, Yes, I've added brackets to my reply but they got removed after posting. :)  
I also tried with brackets before replying and got 500 Server error.

**Reply by ruslandoga** [reply]
Anything in the error logs? `cd hosting && docker compose logs plausible`

**Reply by rafalkukla** [reply]
No specific error, but it does 500 Oops! Looks like we're having server issues
. I enabled LOG_LEVEL=debug but also nothing specific there.

**Reply by wheeleran** [reply]
Did you fix this? I'm looking at it now.

**Comment by parnavh** [comment]
I encounter an error if I enter anything other than a simple email, did anyone find a fix?

**Comment by ruslandoga** [comment]
`MAILER_NAME` has been added in in [v2.0.0-rc.1](https://github.com/plausible/analytics/discussions/2981) (https://github.com/plausible/analytics/pull/2937)
